bin/aapp_dr_runner.py

```diff
@@ -1115,15 +1102,16 @@ def publish_level1(publisher, config, msg, filelist, station_name, environment):
                             renamed_files = rename_aapp_filenames(aapp_config)
                             if not renamed_files:
                                 LOG.warning(
-                                    "The rename of standard aapp filenames to practical ones returned an empty file list")
+                                    "The rename of standard aapp filenames to practical ones " +
+                                    "returned an empty file list")
                                 LOG.warning(
                                     "This means there are no files to publish")
                             else:
                                 publish_level1(
                                     publisher, aapp_config, msg, renamed_files, station_name, environment)
 
                             block_before_rerun(aapp_config, msg)
-                        except Exception as ex:
+                        except Exception:
                             LOG.error("AAPP processing failed.")
                             raise
```

Comment:
Should this use `LOG.exception`?

Reply:
Yeah, probably more correct. Quite a nested code part. But I don't want to change that now...

Reply:
Fixed